lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Transactions.hs

```diff
@@ -1255,47 +1257,60 @@ spec = do
             , expectErrorMessage (errMsg404NoWallet wid)
             ]
 
-    it "SHELLEY_TX_REDEEM_01 - Can redeem rewards from self" $ \ctx -> do
+    -- Reward wallets have 100k ADA UTxO and 1M ADA rewards,
+    -- fixtureWalletsWith have 100k ADA UTxO too
+    -- Verifying withdrawals for some boundary values
+    let rewardsMatrix = [ ("redeem sending exactly rewards", oneMillionAda)
+                        , ("redeem sending exactly UTxO", 100 * oneThousandAda)
+                        , ("redeem sending more than UTxO", 100 * oneThousandAda + oneAda)
+                        , ("redeem sending less than UTxO", 100 * oneThousandAda - oneAda)
+                        , ("redeem sending more than rewards", oneMillionAda + oneAda)
+                        , ("redeem sending less than rewards", oneMillionAda - oneAda)
```

Comment:
Isn't this case redundant with the 2nd and 4th ones :grimacing:  ?

Reply:
I guess not?

Perhaps it's more like 3rd...

Reply:
addressed in https://github.com/input-output-hk/cardano-wallet/pull/1987/commits/ff01561b383b3e0989700fb54a2ff5ae0b12a4da